tests/fixtures/textbox.html

```diff
@@ -0,0 +1,5 @@
+<p>
```

Comment:
Fun fact, if you look at the XML for this, it is actually nested paragraphs. Not really sure how to stop it from doing the outside one. Or even if that will be the correct solution in the general case.